README.md

```diff
@@ -13,7 +13,7 @@ Assuming you have [Rust installed][Rust], run:
 $ cargo install yambar-hyprland-wses
 ```
 
-Use `/home/yourusernamehere/.cargo/bin/yambar-hyprland-wses` as your script path
+Use `/.cargo/bin/yambar-hyprland-wses` as your script path
```

Comment:
Ah, this isn't quite right. You need the `~` at the beginning of the path to indicate the home directory, and the same in the YAML down below.

Reply:
Wow, I swear I added `~` at the front :sweat_smile:

I'll get that sorted.